### DISCUSSION TITLE
Incomplete torrents error with Nohardlink

### DISCUSSION THREAD

**Discussion by MrMarble** [StuffAnThings/qbit_manage#220] [Q&A]
Hi, I have qBittorrent running on docker with multiple volumes attached.
Ongoing downloads (aka incomplete) have the path set to `/incomplete` and on finish, they move to a different path mounted on my NAS `/media/Downloads/qbittorrent/complete/downloads`, this path changes in relation with the category :`/media/Downloads/qbittorrent/complete/radarr`, `/media/Downloads/qbittorrent/complete/sonarr`...

This is my config for the directories on qbit_manage:
```yaml
directory:
  root_dir: "/media/Downloads/qbittorrent/complete"
```

This seems to work for the most part, but sometimes qbit_manage runs while there are torrents downloading and have the path set to `/incomplete` giving me this error and crashing:
```
|================================ Tagging Torrents with No Hardlinks ================================|
| Nohardlink Error: Unable to open the folder /incomplete/Some-legally-downloaded-movie.mkv. Please make sure folder exists and qbit_manage has access to this directory. |
| Uncaught Exception                                                                                 |
Traceback (most recent call last):
  File "/app/qbit_manage.py", line 533, in <module>
    start_loop()
  File "/app/qbit_manage.py", line 308, in start_loop
    start()
  File "/app/qbit_manage.py", line 402, in start
    num_tagged, num_untagged, num_deleted, num_deleted_contents = cfg.qbt.tag_nohardlinks()
  File "/app/modules/qbittorrent.py", line 521, in tag_nohardlinks
    has_nohardlinks = util.nohardlink(torrent["content_path"].replace(root_dir, remote_dir), self.config.notify)
  File "/app/modules/util.py", line 291, in nohardlink
    largest_file_size = os.stat(sorted_files[0]).st_size
IndexError: list index out of range
```

I this an issue with qbit_manage, or I need to configure/change something.

I have qbit_manage version v3.4.2

Thanks!

**Comment by bakerboy448** [comment]
Still an issue?
